ddl/backfilling.go

```diff
@@ -425,6 +425,13 @@ func (w *worker) handleReorgTasks(reorgInfo *reorgInfo, totalAddedCount *int64,
 }
 
 func tryDecodeToHandleString(key kv.Key) string {
+	defer func() {
+		if r := recover(); r != nil {
```

Comment:
Could we use `tidbutil.Recover` instead of this code?

Reply:
But there is no way to print the value of `key`.

If we use `tidbutil.Recover()` and pass a function as the argument, there is no difference from the current implementation.

Reply:
Got it.